Emby.Server.Implementations/ScheduledTasks/TaskManager.cs

```diff
@@ -49,7 +55,7 @@ public TaskManager(
         /// Gets the list of Scheduled Tasks.
         /// </summary>
         /// <value>The scheduled tasks.</value>
-        public IScheduledTaskWorker[] ScheduledTasks { get; private set; }
+        public IReadOnlyCollection<IScheduledTaskWorker> ScheduledTasks { get; private set; }
```

Comment:
Any reason to use `IReadOnlyCollection` over `IReadOnlyList`? PR currently uses both